package.json

```diff
@@ -23,7 +23,7 @@
   "devDependencies": {
     "backbone": "1.2.1",
     "chai": "^3.0.0",
-    "karma": "^0.12.36",
+    "karma": "^0.13.9",
```

Comment:
This was motivated by karma failing to properly load the webpack module when running npm install && npm run-script test on a fresh checkout, which is a bug mentioned elsewhere that's fixed by upgrading karma.